tutorials/circuits/1_getting_started_with_qiskit.ipynb

```diff
@@ -6,39 +6,26 @@
    "source": [
```

Comment:
<p>Maybe we don't need to explain the resulting state of each step</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Maybe not strictly necessary but I thought it was a nice detail that didn't divert too much from the steps. I'm open to removing these parts but the rest (explaining which qubits the gate acts on) I'd keep.</p>